docs/Documentation/model_introduction.md

```diff
@@ -10,22 +10,46 @@
 
-Conversion is not required for ONNX, PaddlePaddle, TensorFlow Lite and TensorFlow models (check :doc:`TensorFlow Frontend Capabilities and Limitations <openvino_docs_MO_DG_TensorFlow_Frontend>`), as OpenVINO provides C++ and Python APIs for importing them to OpenVINO Runtime directly. It provides a convenient way to quickly switch from framework-based code to OpenVINO-based code in your inference application.
+Convert a model in Python
+######################################
+
+Model conversion API, specifically, the ``mo.convert_model()`` method converts a model from original framework to ``ov.Model``. ``mo.convert_model()`` returns ``ov.Model`` object in memory so the ``read_model()`` method is not required. The resulting ``ov.Model`` can be inferred in the same training environment (python script or Jupiter Notebook). ``mo.convert_model()`` provides a convenient way to quickly switch from framework-based code to OpenVINO-based code in your inference application. In addition to model files, ``mo.convert_model()`` can take OpenVINO extension objects constructed directly in Python for easier conversion of operations that are not supported in OpenVINO. The ``mo.convert_model()`` method also has a set of parameters to :doc:`cut the model <openvino_docs_MO_DG_prepare_model_convert_model_Cutting_Model>`, :doc:`set input shapes or layout <openvino_docs_MO_DG_prepare_model_convert_model_Converting_Model>`, :doc:`add preprocessing <openvino_docs_MO_DG_Additional_Optimization_Use_Cases>`, etc.
+
+.. image:: _static/images/model_conversion_diagram.svg
+   :alt: model conversion diagram
+
+Convert a model with ``mo`` command-line tool
+#############################################
+
+Another option to convert a model is to use ``mo`` command-line tool. ``mo`` is a cross-platform tool that facilitates the transition between training and deployment environments, performs static model analysis, and adjusts deep learning models for optimal execution on end-point target devices in the same measure, as the ``mo.convert_model`` method.
+
+``mo`` requires the use of a pre-trained deep learning model in one of the supported formats: TensorFlow, TensorFlow Lite, PaddlePaddle, or ONNX. ``mo`` converts the model to the OpenVINO Intermediate Representation format (IR), which needs to be read with the ``ov.read_model()`` method. Then, you can compile and infer the ``ov.Model`` later with :doc:`OpenVINO™ Runtime <openvino_docs_OV_UG_OV_Runtime_User_Guide>`.
+
+
+The figure below illustrates the typical workflow for deploying a trained deep learning model:
+
+.. image:: _static/images/BASIC_FLOW_MO_simplified.svg
```

Comment:
![image](https://github.com/openvinotoolkit/openvino/assets/29092471/596d2223-e5c8-46c5-8d9e-767ae2064bdc)

Reply:
For future improvements (I believe you will apply it in a short-term): put examples for both flows near the picture to clearly see in the code how this flow is implemented. It shouldn't be exhaustive examples, but something that is correctly calls highlighted functions on the pictures.

Reply:
The changes have been implemented, see the commit: [21dd5fa](https://github.com/openvinotoolkit/openvino/pull/17454/commits/21dd5fa81eccd8564134ea5a65896555791a0e68)